releaseNotes/Microsoft.Featuremanagement.Telemetry.ApplicationInsights.md

```diff
@@ -2,6 +2,26 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0-preview4 - Jul 19, 2024
+
+* Updated `Microsoft.FeatureManagement` reference to `4.0.0-preview4`. See the [release notes](./Microsoft.Featuremanagement.md) for more information on the changes.
+
+### Enhancements
+
+* Added new `AddApplicationInsightsTelemetryPublisher` API to register a telemetry publisher to send events to Application Insights. [#455](https://github.com/microsoft/FeatureManagement-Dotnet/pull/455)
+
+``` C#
+builder.Services.AddFeatureManagement()
+    .WithTargeting()
+    .AddApplicationInsightsTelemetryPublisher();
+```
```

Comment:
NIT: add indentation to the code snippet to align with the bullet points.

Reply:
The way to add indentation is to add double empty spaces to each line of the code snippet in case you are not sure.

Reply:
Oh, sorry. I misunderstand where to add indentation. Now I get your point.